tests/std/tests/P2474R2_views_repeat/test.cpp

```diff
@@ -360,6 +360,16 @@ constexpr bool test() {
     test_iterator_arithmetic<wchar_t>();
     test_iterator_arithmetic<_Signed128>();
 
+    // GH-4507: LWG-4053 Unary call to std::views::repeat does not decay the argument
+    {
+        using RPV = std::ranges::repeat_view<const char*>;
+
+        static_assert(std::same_as<decltype(std::views::repeat("foo", std::unreachable_sentinel)), RPV>);
+        static_assert(std::same_as<decltype(std::views::repeat(+"foo", std::unreachable_sentinel)), RPV>);
+        static_assert(std::same_as<decltype(std::views::repeat("foo")), RPV>);
+        static_assert(std::same_as<decltype(std::views::repeat(+"foo")), RPV>);

```

Comment:
`using namespace std;` is already in this test file, so we should conventionally drop all `std::` here.

Also, LWG-4053 should be mentioned in the title of this PR.